### DISCUSSION TITLE
Textual Inversion in kubin

### DISCUSSION THREAD

**Discussion by videoprice** [seruva19/kubin#17] [General]
Hello, guys I use this script
https://github.com/TheDenk/Kandinsky-2-textual-inversion/
and for result got this file: learned_embeds.bin
Don't know whats next:) I have a merge this bin with \models\2_1\text_encoder\pytorch_model.bin
Can something explain here, where is point:)
Thanks guys

**Comment by adrianpuiu** [comment]
open the jupyter notebook that says inference textual inversion and modify the path to learned_embeds.bin 

if you don't have the jupyter-lab installed install it via pip. : `pip install jupyterlab` and launch  with : jupyter-lab then open the inference ti notebook

**Reply by videoprice** [reply]
thank a lot, works. :)

**Reply by adrianpuiu** [reply]
can you share your model ? what is it about ? how to trigger ?

**Reply by adrianpuiu** [reply]
https://drive.google.com/file/d/1BkI4sBrukPTKFdp3Y72WnwFOqLdRaypW/view?usp=sharing , try my textual inversion...  placeholder_token=>sks
initializer_token=>painting

**Reply by videoprice** [reply]
sorry I cant get this my first model, because it's my family:) But in a future I can train something new
